# function printing error
def error_wrong_square(self):
    print("Niepoprawne dane, wprowadz wspolrzedne jeszcze raz:" + "\n")
    self.test()
